bdk-ffi/src/keys.rs

```diff
@@ -220,14 +253,17 @@ impl DescriptorPublicKey {
         }
     }
 
+    /// Convert the descriptor to a string.
     pub fn as_string(&self) -> String {
         self.0.to_string()
     }
```

Comment:
Same as above: this is a vestige of the times where we could not get the Display trait on types.

I think we should add the Display in this case so the `.toString()` method is implemented for `DescriptorPublicKey`.